index.html

```diff
@@ -469,6 +471,8 @@ <h3>Bagit Profile Json file</h3>
         bag-requirements-section              = manifests-required
                                                 manifests-allowed
                                                 allow-fetch
+                                                fetch.txt-required
```

Comment:
ABNF rule names can't contain periods. I'd suggest "fetch-required" here.

Also, each of these new rules need to be expressed below (see subsequent comment).